### PR TITLE
help pages for provider and advocate

### PR DESCRIPTION
**Pull request by nobane**

### CODE REVIEW

File: _15thnight/api/account.py [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-    from config import RESET_TOKEN_LIFE
-except:
-    from config import RESET_TOKEN_LIFE
+from _15thnight.util import api_error, jsonify, required_access
 
 
 account_api = Blueprint('account_api', __name__)
 
-reset_token_life = timedelta(hours=RESET_TOKEN_LIFE)
+reset_token_life = timedelta(hours=current_app.config.get('RESET_TOKEN_LIFE', 24))
```
Comment (justinnoah) [inline _15thnight/api/account.py]: current_app is not imported

File: _15thnight/core.py [hunk 2]
```diff
@@ -130,3 +136,56 @@ def send_out_resolution(need):
...
+    queue_send_email.apply_async(kwargs=dict(message=message))
+
+
+def send_confirm_password_reset(user):
+    """
+    Send confirmation of password reset.
+    """
+    data = dict(
+        time=datetime.utcnow().strftime('%m/%d/%y %I:%M%p'),
+        ip=request.remote_addr
```
Comment (justinnoah) [inline _15thnight/core.py:166]: request is not defined

**Comment by nobane** [comment]
@justinnoah addressed your feedback

File: _15thnight/core.py [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+from flask_mail import Message
 
-from _15thnight.queue import queue_send_message
+from _15thnight.queue import queue_send_message, queue_send_email
 from _15thnight.models import (
     Alert, Need, NeedProvided, ProviderNotified, Service, Response, User
 )
 
+reset_token_life = timedelta(
+    hours=current_app.config.get('RESET_TOKEN_LIFE', 24))
```
Comment (justinnoah) [inline _15thnight/core.py]: RuntimeError: working outside of application context

Move this into send_password_reset. It's the only method that currently uses it and is always called within an application context

File: _15thnight/api/account.py [hunk 4]
```diff
@@ -1,27 +1,24 @@
...
-    from config import RESET_TOKEN_LIFE
-except:
-    from config import RESET_TOKEN_LIFE
+from _15thnight.util import api_error, jsonify, required_access
 
 
 account_api = Blueprint('account_api', __name__)
 
-reset_token_life = timedelta(hours=RESET_TOKEN_LIFE)
+reset_token_life = timedelta(
```
Comment (justinnoah) [inline _15thnight/api/account.py]: This causes a runtime error

define this in reset_password. it is the only method that uses it and is always called within an application context.

**Comment by justinnoah** [comment]
Receiving a 405 when attempting to login. Either as admin, provider, or advocate.

**Comment by nobane** [comment]
@justinnoah sounds like your client is out of date.

**Comment by justinnoah** [comment]
yep, forgot to run webpack.

**Comment by justinnoah** [comment]
Looks good and works. Merging.

**Comment by justinnoah** [comment]
@spectralsun This closes #152 doesn't it? If so, can you make a note on the issue that due to this PR, it is to be closed, and close the issue? Thanks!

**Pull request merged**